packages/map/src/components/WorldMap/WorldMap.tsx

```diff
@@ -191,12 +190,7 @@ const WorldMap = () => {
     <ErrorBoundary component='WorldMap'>
       {allowMapZoom ? (
         <svg viewBox={SVG_VIEWBOX} role='img' aria-label={handleMapAriaLabels(config)}>
-          <rect
-            height={SVG_HEIGHT}
-            width={SVG_WIDTH}
-            onClick={() => handleReset(config, setRuntimeData)}
-            fill='white'
-          />
+          <rect height={SVG_HEIGHT} width={SVG_WIDTH} onClick={() => handleReset(setRuntimeData)} fill='white' />
```

Comment:
While we're here can we refactor this so that we aren't currying the setRuntimeData function for no reason?